doc/examples/transform/plot_ransac.py

```diff
@@ -114,3 +117,53 @@
            marker='o', label='Outlier data')
 ax.legend(loc='lower left')
 plt.show()
+
+######################################################################
+# Circle model
+# ============
+from skimage.measure import CircleModel
+
+
+def generate_noisy_circle(size, noise_level, radius=1):
+    angle = 2 * np.pi * np.random.rand(size)
+    rad = radius + noise_level * (2 * np.random.rand(size) - 1)
+    x = rad * np.cos(angle)
+    y = rad * np.sin(angle)
+    return x, y
+
+
+# Generate three types of data points
```

Comment:
Can you briefly describer what the purpose of these 3 are and which one we are expecting to estimate with the circle model?